### PR TITLE
GraphQL timeline

### PR DESCRIPTION
**Pull request by zedeus**

### CODE REVIEW

File: src/consts.nim [hunk 1]
```diff
@@ -19,8 +19,10 @@ const
   tweet* = timelineApi / "conversation"
 
   graphql = api / "graphql"
+  graphUserTweets* = graphql / "9rys0A7w1EyqVd2ME0QCJg/UserTweets"
+  graphUserTweetsAndReplies* = graphql / "ehMCHF3Mkgjsfz_aImqOsg/UserTweetsAndReplies"
   graphTweet* = graphql / "6lWNh96EXDJCXl05SAtn_g/TweetDetail"
-  graphUser* = graphql / "7mjxD3-C6BxitPMVQ6w0-Q/UserByScreenName"
+  graphUser* = graphql / "nZjSkpOpSL5rWyIVdsKeLA/UserByScreenName"
```
Comment (markcellus) [inline src/consts.nim:25]: May I ask, where are these values coming from? Are you seeing these show up in the network calls? If so, are they ephemeral?

Comment (zedeus) [reply src/consts.nim:25]: It's some sort of versioning ID that gives you persistent access to a specific version of the API. It's probably random, haven't tried to decode it, but you can find plenty of versions by searching for the endpoints on GitHub and elsewhere.

Comment (zedeus) [reply src/consts.nim:25]: You can get the newest one by opening devtools, going to the network tab, filtering by graphql, then loading pages on Twitter.

**Comment by samtin0x** [comment]
Is everything set to merge this PR? thanks!

**Pull request merged**